contrib/init/sysvinit-redhat/docker

```diff
@@ -56,7 +56,7 @@ start() {
         prestart
         printf "Starting $prog:\t"
         echo "\n$(date)\n" >> $logfile
-        "$unshare" -m -- $exec daemon $other_args &>> $logfile &
+        "$unshare" -m -- $exec daemon $other_args & >> $logfile &
```

Comment:
Does this still work with a space? https://www.gnu.org/software/bash/manual/bashref.html#Appending-Standard-Output-and-Standard-Error doesn't say much about whitespace, but I'd imagine the space is necessary, so I'm curious whether this will still have the correct effect